src/CMakeLists.txt

```diff
@@ -1287,6 +1289,7 @@ add_custom_target(vstart DEPENDS
 add_custom_target(cephfs_testing DEPENDS
     vstart
     rados
+    cython_modules
```

Comment:
Nothing we actually _build_ depends on them, right?  So it would only be testing-related targets that will run using the modules that would have the dependency, and then presumably packaging one day.